.travis.yml

```diff
@@ -1,6 +1,10 @@
+sudo: required
+services:
+  - docker
 language: node_js
 env:
   - CXX=g++-4.8
+before_install: if [[ `npm -v` != 3* ]]; then npm i -g npm@3; fi
```

Comment:
I actually ended up not needing this because at one point in time I needed npm3 installed to deal with a bundledDependency.

So now it's stock on both, which is even more disparate but justifiable? also how it used to be.